### PR TITLE
Create views for Employees

### LINKED ISSUE #27: Create views for Employees
# REQUIREMENTS
We're going to create the views for the employees controller

# Acceptance Criteria
List all required acceptance criteria
- [ ] Create the views for the crud

### PR DESCRIPTION
**Pull request by SEVERO96**
what? / why?
The view was added to show all the records of the employees and actions to add and edit to be able to interact with the dashboard

image

### CODE REVIEW

File: checador/app/views/employees/employee.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <td>Juan</td>
+          <td>juanbonix@gmail.com</td>
+          <td>Gerente</td>
+          <td>2</td>
+          <td>23452</td>
+          <td>Active</td>
+          <td>
+            <%= link_to new_employee_path, class: 'btn btn-primary' do %>
+             <i >EDIT</i>
+                 <% end %>
```
Comment (xFerchoVJ) [inline checador/app/views/employees/employee.html.erb]: You can also do
`<%= link_to "EDIT", new_employee_path, class:"btn btn-primary"%>`

**Review by SampabloC — DISMISSED** [review]

**Review by xFerchoVJ — DISMISSED** [review]

File: checador/app/views/employees/employee.html.erb (hunk 1 above)
Comment (SEVERO96) [reply checador/app/views/employees/employee.html.erb]: ok.

**Review by martinmt-mx — APPROVED** [review]
nice!

**Review by xFerchoVJ — APPROVED** [review]
LGTM!

**Review by SaraNavarro — APPROVED** [review]
Nice View.

**Pull request merged**